src/declarations.ts

```diff
@@ -135,5 +137,15 @@ declare module 'fastify' {
     h5p?: {
       taskManager: H5PTaskManager;
     };
+
+    /**
+     * admin service
+     */
+    admin?: {
+      dbService: AdminService;
+      taskManager: AdminTaskManager;
+      adminRoleId: string;
+      verifyIsAdmin: (request: FastifyRequest, reply: FastifyReply) => boolean;
```

Comment:
Why does the return value depend on both the request and the reply? Is this the right input level of abstraction (i.e., can and should any request-reply pair be checked against this) ?